doc/en/mooncake-store.md

```diff
@@ -488,6 +488,18 @@ There are two startup parameters in `master_service` related to the soft pin mec
 
 Notably, soft pinned objects can still be removed using APIs such as `Remove` or `RemoveAll`.
 
+### Zombie Object Cleanup
```

Comment:
The document is good and clean. Could you also  copy-paste to the docs/source/design/mooncake-store.md as well?